libc/CMakeLists.txt

```diff
@@ -17,9 +17,9 @@ if (OE_SGX)
     ${MUSLSRC}/math/x86_64/log2l.s
     ${MUSLSRC}/math/x86_64/logl.s
     #${MUSLSRC}/math/x86_64/exp2l.s
-    ${MUSLSRC}/math/x86_64/sqrt.s
-    ${MUSLSRC}/math/x86_64/sqrtl.s
-    ${MUSLSRC}/math/x86_64/sqrtf.s
+    ${MUSLSRC}/math/x86_64/sqrt.c
```

Comment:
Consider moving the .c files out of the asm library.

Reply:
These arch specific files are grouped right now together. It would mean break these up if I move it to another OBJECT in the CMakeLists.txt file.. I prefer keeping these here..

Reply:
Moved .c files from libasm to PLATFORM_SRC